lib/modules/auth/login/bloc/login_event.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:shop_app/models/login/login_model.dart';
+
+abstract class LoginEvent{}
+
+class UserLoginEvent extends LoginEvent{
+  LoginModel loginModel;
+
+  UserLoginEvent(this.loginModel);
+}
+
+class ChangePasswordVisibility extends LoginEvent{
```

Comment:
change class name to  ChangePasswordVisibilityEvent

Reply:
done